lms/urls.py

```diff
@@ -429,6 +429,12 @@
             include('edxnotes.urls'), name="edxnotes_endpoints"),
     )
 
+    if settings.FEATURES["ENABLE_TEAMS"]:
```

Comment:
I would prefer to make that a separate story, if we choose to do it. This feature flag is not new with this PR.